src/apps/stable/routes/plexbot/PaginationAction.tsx

```diff
@@ -0,0 +1,76 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable react/jsx-no-bind */
+
+import React from 'react';
+
+import { KeyboardArrowLeft, KeyboardArrowRight } from '@mui/icons-material';
+import { Box, IconButton } from '@mui/material';
+import FirstPageIcon from '@mui/icons-material/FirstPage';
+import LastPageIcon from '@mui/icons-material/LastPage';
+
+interface TablePaginationActionsProps {
+    count: number;
```

Comment:
count is the total pages count ? if that's the case rename it to totalPages for more clarity

Reply:
These are Props sent by Mui i can't change them please look at line 110 in plexbotDashboard